test/.jshintrc

```diff
@@ -19,6 +19,9 @@
 	"devel": true,
 	"wsh": true,
 
+	// Support: New integer literals
+	"esnext": true,
```

Comment:
This would enable esnext for all the source code and unit tests, I don't think we want to do that because the source should be plain ES5. Can we just enable for the one test where we need it?